cocos2dx/base_nodes/CCAtlasNode.h

```diff
@@ -102,15 +102,15 @@ class CC_DLL CCAtlasNode : public CCNode, public CCRGBAProtocol, public CCTextur
     */
     virtual void updateAtlasValues();
 
-    virtual void draw(void);
+    virtual void draw(void) override;
```

Comment:
Did we want to keep both "virtual" and "override" keywords here? Not that it matters that much I don't think.